### ISSUE TITLE
`CS8400: Feature 'interpolated string handlers` on building with net6.0 and LangVersion 8.0

### ISSUE THREAD

**Issue by jeromelaban** [dotnet/roslyn#55564]
**Version Used**: Version 17.0.0 Preview 3.0

**Steps to Reproduce**:

Compile the following code:
```csharp
class Test01
{
	static void C()
	{
		var uri = new System.Text.StringBuilder();
		byte b = 0x0;
		uri.Append($"%{b:X2}");
	}
}
```
With the following project configuration:
```xml
  <PropertyGroup>
    <TargetFramework>net6.0</TargetFramework>
    <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
    <Nullable>enable</Nullable>
    <LangVersion>8.0</LangVersion>
  </PropertyGroup>
```

**Expected Behavior**: 
The code compiles

**Actual Behavior**: 
```
error CS8400: Feature 'interpolated string handlers' is not available in C# 8.0. Please use language version 10.0 or greater.
```

**Comment by reflectronic** [comment]
This is fixed by #55380.

**Comment by svick** [comment]
Closing as duplicate of https://github.com/dotnet/roslyn/issues/55345 (the issue that was fixed by https://github.com/dotnet/roslyn/pull/55380).
